src/ol_infrastructure/substructure/vault/setup/Pulumi.substructure.vault.setup.operations.CI.yaml

```diff
@@ -0,0 +1,5 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-ci
+encryptedkey: AQICAHjEYf3gV5ZTnviCJcHwGnUuLzld5EuYumgMrRde6bvXUgG0NuGxuciUOUHhP1tZyHF3AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9bsC9GV4ySfLwLmOAgEQgDuezZ5q7HcL7ipEnXE1Gvg2QqqTozAo8eJ7CIyhK75F1jblIPgmVLfHIC2ZFyOtpb+doYAYNUH+erzcDw==
+config:
```

Comment:
Does the same info needs to live here and in the auth substructure?

Reply:
Yes, there's not a way to share info across stacks unfortunately. We'll have to add our own method for cross-cutting values.